edit/monitoring.coreos.com.receiver/types/opsgenie.vue

```diff
@@ -138,11 +143,11 @@ export default {
         <h3>Responders</h3>
         <ArrayList v-model="responders" :mode="mode" :default-add-value="defaultResponder" :show-header="true">
           <template v-slot:column-headers>
-            <div class="row mb-10">
+            <div class="row" :class="{'mb-15': isView}">
```

Comment:
Looks better with the mb-10 left on when not in view mode - can we add that as well